swagger.json

```diff
@@ -1121,6 +1121,122 @@
+                    {
+                      "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                      "description": "ID (integer) of the driver with HOS logs.",
+                      "in": "path",
+                      "name": "driver_id",
+                      "required": true,
+                      "type": "string"
+                    },
+                    {
+                        "description": "A specified timestamp within the bounds of a driver's certified day chart, specified in milliseconds UNIX time.",
+                        "in": "path",
+                        "name": "time_ms",
+                        "required": true,
+                        "type": "integer"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns a certified day chart for the driver, if one exists",
+                        "schema": {
+                            "$ref": "#/definitions/HosCertifiedDayChartResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+              "summary": "/fleet/drivers/{driver_id:[0-9]+}/hos_certified_day_chart/{time_ms:[0-9]+}",
+              "description": "Modify details, such as trailer number or shipping ID, for a driver's certified day chart",
+              "operationId": "patch_hos_certified_day_chart",
+              "tags": [
+                "Fleet", "Drivers"
+              ],
+              "produces": [
+                "application/json"
+              ],
+              "consumes": [
+                "application/json"
+              ],
+              "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "description": "ID (integer) of the driver with certified day chart.",
+                  "in": "path",
+                  "name": "driver_id",
+                  "required": true,
+                  "type": "integer"
+                },
+                {
+                    "description": "A specified timestamp within the bounds of a driver's certified day chart, specified in milliseconds UNIX time.",
+                    "in": "path",
+                    "name": "time_ms",
+                    "required": true,
+                    "type": "integer"
+                },
+                {
+                  "in": "body",
+                  "name": "hosCertifiedDayChartParams",
+                  "required": true,
+                  "schema": {
+                    "properties": {
+                      "shippingDocNumbers": {
```

Comment:
Would `Id` make sense over `Number` here? We don't define what their shipping doc should look like, so it could be alphanumeric.

Reply:
The official term is `Shipping Document Number` from the legal lingo which is why I was chose those words here. Do you still think Id is more appropriate?

Reply:
Do we expect this to always be a number? I think we should update our legal lingo since it's incorrect :)

Reply:
Sorry, I meant specifically from the ELD mandate: https://www.law.cornell.edu/cfr/text/49/appendix-A_to_subpart_B_of_part_395